.github/workflows/docs.yml

```diff
@@ -0,0 +1,15 @@
+name: docs
+on:
+  push:
+    branches:
+      - st4000-exploration
```

Comment:
I guess the docs only needs a rebuild when something in `docs/` is pushed?
https://github.blog/changelog/2019-09-30-github-actions-event-filtering-updates/

Reply:
So if I write:
```
on:
  push:
    paths:
    - "docs/**"
```
this github actions will be triggered when a file in the docs folder is changed/added/deleted?

Reply:
Yes. And the workflow file itself. https://github.com/jfcherng-sublime/ST-ASS/blob/0db07aac67609f0731374601a1557b7f4723c2d5/.github/workflows/syntax.yml#L3-L15

Reply:
Thanks :)
done with 2f2f369

Reply:
You forget to add `.github/workflows/docs.yml`!

Reply:
done 57b67eb :)